src/vabackend.c

```diff
@@ -38,6 +39,13 @@
 #define gettid pthread_getthreadid_np
 #endif
 
+#ifndef gettid
```

Comment:
Does it really work? Can we check if the function is defined by using `#ifndef`?